kaitai-struct-visualizer.gemspec

```diff
@@ -34,4 +34,5 @@ Gem::Specification.new do |s|
   # s.add_development_dependency 'rspec', '~> 3'
 
   s.add_dependency 'kaitai-struct', '~> 0.4'
+  s.add_dependency 'win32-api', '~> 1.10.1'
```

Comment:
Adding this unconditionally makes it impossible to compile this on anywhere but Windows:

```
$ gem install kaitai-struct-visualizer-0.7.gem
Fetching win32-api-1.10.1.gem
Fetching kaitai-struct-0.9.gem
Building native extensions. This could take a while...
ERROR:  Error installing kaitai-struct-visualizer-0.7.gem:
        ERROR: Failed to build gem native extension.
```

and the culprit being:

```
"x86_64-w64-mingw32-gcc -o conftest.exe -IC:/Ruby30-x64/include/ruby-3.0.0/x64-mingw32 -IC:/Ruby30-x64/include/ruby-3.0.0/ruby/backward -IC:/Ruby30-x64/include/ruby-3.0.0 -I. -D__USE_MINGW_ANSI_STDIO=1 -DFD_SETSIZE=2048 -D_WIN32_WINNT=0x0600 -D__MINGW_USE_VC2005_COMPAT -D_FILE_OFFSET_BITS=64  -O3 -fno-fast-math -fstack-protector-strong -fno-omit-frame-pointer conftest.c  -L. -LC:/Ruby30-x64/lib -L. -pipe -s -fstack-protector-strong  -m64   -lx64-msvcrt-ruby300  -lshell32 -lws2_32 -liphlpapi -limagehlp -lshlwapi  "
x86_64-w64-mingw32-gcc: error: unrecognized command line option '-fstack-protector-strong'
x86_64-w64-mingw32-gcc: error: unrecognized command line option '-fstack-protector-strong'
checked program was:
/* begin */
1: #include "ruby.h"
2: 
3: #include <winsock2.h>
4: #include <windows.h>
5: int main(int argc, char **argv)
6: {
7:   return !!argv[argc];
8: }
/* end */
```

... which is might scary by itself even on Windows, as this literally starts running compiler with arbitrary hardcoded paths such as `C:/Ruby30-x64`.

Reply:
No longer dependent on 'win32-api'